website/docs/docs/build/environment-variables.md

```diff
@@ -118,6 +118,7 @@ The following environment variables are set automatically:
 - `DBT_ENV` &mdash; This key is reserved for the dbt Cloud application and will always resolve to 'prod'. For deployment runs only.
 - `DBT_CLOUD_ENVIRONMENT_NAME` &mdash; The name of the dbt Cloud environment in which `dbt` is running. 
 - `DBT_CLOUD_ENVIRONMENT_TYPE` &mdash; The type of dbt Cloud environment in which `dbt` is running. The valid values are `dev`, `staging`, or `prod`. It can be unset, so use a default like `{{env_var('DBT_CLOUD_ENVIRONMENT_TYPE', '')}}`.
+- `DBT_CLOUD_INVOCATION_CONTEXT` &mdash; The context type of the dbt Cloud environment in which `dbt` is invoked. The values are `dev`, `staging`, `prod`, or `ci`. 
```

Comment:
ooh wait i might trim this to just be 

the context type ~of the dbt Cloud environment~ in which `dbt` is invoked